src/ingestor.ts

```diff
@@ -79,6 +80,16 @@ export default async function ingestor(req) {
     if (!Object.keys(hashTypes).includes(hash)) return Promise.reject('wrong types');
     type = hashTypes[hash];
 
+    try {
+      if (
+        (message.space.startsWith('s:') || !message.space.includes(':')) &&
+        ensNormalize(message.space) !== message.space.toLowerCase()
```

Comment:
I think v2 should send requests with just space id without the `s:` prefix

Reply:
We can't, as it's required to differentiate different networks.

Sequencer also accepts space id like `sn:0x0000` and `sep:0x000`